src/templates/select.hbs

```diff
@@ -30,8 +30,16 @@
     {{#if isLoading}}
       <span class="ember-select-loading">Loading...</span>
     {{/if}}
-    {{#if hasNoResults}}
-      <span class="ember-select-no-results">No results match "{{query}}"</span>
+    {{#if contentIsEmpty}}
+      <div class="ember-select-empty-content">
+        {{#if emptyContentView}}
```

Comment:
Yeah I thought it's possible to render empty view so the test might be useless. Okay updated.